src/app/templates/app.aside.html

```diff
@@ -1,10 +1,31 @@
 <div class="aside story">
-    <h2 tabIndex="0">Full index</h2>
-    <div class="status info" ng-show="!vm.selectedIndex.resources">
-        Select an index to see download links
+
+    <div id="ssq-minicart">
+
+        <h2>Cart</h2>
+
+        <div id="ssq-minicart-view">
+            <!-- TODO noscript customer instructions -->
+            <p class="ssq-minicart-noscript">Edit cart or checkout to place your order.</p>
+
+            <div class="ssq-minicart-submit">
+                <input type="hidden" id="ssq-cart-contents" name="ssq-cart-contents" value=""/>
+                <a href="https://test.smartservice.qld.gov.au/payment/cart/checkout"><img id="ssq_minicart_checkout"
+                                                                  src="https://test.smartservice.qld.gov.au/payment/minicart/btn-checkout.png"
+                                                                  alt="Checkout"/></a>
+                <a href="https://test.smartservice.qld.gov.au/payment/cart/view"><img id="ssq_minicart_cart"
+                                                              src="https://test.smartservice.qld.gov.au/payment/minicart/btn-cart.png"
+                                                              alt="Edit cart"/></a>
+            </div>
+        </div>
+
+        <div class="ssq-minicart-cards">
+            <h3>Cards accepted</h3>
+            <ul>
+                <li><img src="https://test.smartservice.qld.gov.au/payment/minicart/visa.png" alt="Visa"/></li>
```

Comment:
would html hardcoding also be handled?

Reply:
Yes. That part has been amended in the way mentioned [here](https://github.com/qld-gov-au/qsa-archive-search/pull/3#discussion_r143388164). The file is not going to have `https://test.smartservice.qld.gov.au` hard-coded as follows:

```html
(Omitting)
            <div class="ssq-minicart-submit">
                <input type="hidden" id="ssq-cart-contents" name="ssq-cart-contents" value=""/>
                <a href="{{vm.EnvService.smartServiceUrlPrefix}}/payment/cart/checkout"><img id="ssq_minicart_checkout"
                                                                  src="{{vm.EnvService.smartServiceUrlPrefix}}/payment/minicart/btn-checkout.png"
                                                                  alt="Checkout"/></a>
                <a href="{{vm.EnvService.smartServiceUrlPrefix}}/payment/cart/view"><img id="ssq_minicart_cart"
                                                              src="{{vm.EnvService.smartServiceUrlPrefix}}/payment/minicart/btn-cart.png"
                                                              alt="Edit cart"/></a>
(Omitting)
        <div class="ssq-minicart-cards">
            <h3>Cards accepted</h3>
            <ul>
                <li><img src="{{vm.EnvService.smartServiceUrlPrefix}}/payment/minicart/visa.png" alt="Visa"/></li>
                <li><img src="{{vm.EnvService.smartServiceUrlPrefix}}/payment/minicart/mastercard.png" alt="MasterCard"/></li>
(Omitting)
```